e2e/keywords/common.resource

```diff
@@ -26,6 +27,7 @@ Set test environment
     set_backupstore
 
 Cleanup test resources
+    power_on_all_nodes
```

Comment:
Is this still required? It introduces extra dependencies to the cleanup function. It would be great if we can figure out a better cleanup mechanism and avoid adding everything to this cleanup function. Do you have any idea?

Reply:
Updated in common.resouce as below

```
Cleanup test resources include off nodes
    Power on off node
```